docs/ROADMAP.md

```diff
@@ -0,0 +1,117 @@
+## Q3 Roadmap - Tentative
+
+### Overview
+
+---
+
+In Q3, there are a few groups of tasks we'd like to work on:
+
+- **Continue with the stories from Q2**: Authentication, Authorization,
+  Inclusion of Related models
+- **Feature parity gap**: Declarative support
+- **Internal tooling/infrastructure**: support Node.js 12 in juggler &
+  connectors, reduce build time, etc.
+- **Juggler-next**: some groundwork to prepare us to work on juggler-next, e.g.
+  spike on how the code resides affect our build process
+
+### Scope
+
+---
+
+#### Authentication
+
+- [ ] Token based authentication in API Explorer.
+      [Spike](https://github.com/strongloop/loopback-next/issues/2027)
+- [ ] Stretch goal: enable class level `@authenticate` decorator:
+      https://github.com/strongloop/loopback-next/issues/2460
+
+#### Authorization
+
+- [ ] Common layer for authentication & authorization
+      https://github.com/strongloop/loopback-next/issues/2900
```

Comment:
Maybe we should finish https://github.com/strongloop/loopback-next/pull/1205 first as it will help us find out what common types/interfaces have to be shared between authentication and authorization.

Reply:
updated